Modelica/Magnetic/FluxTubes/Examples/BasicExamples/ToroidalCoreAirgap.mo

```diff
@@ -0,0 +1,118 @@
+        extent={{-10,-10},{10,10}},
+        rotation=90,
+        origin={-60,0})));
+  Modelica.Magnetic.FluxTubes.Sensors.MagneticFluxSensor magFluxSensor
+    annotation (Placement(transformation(
+        extent={{10,-10},{-10,10}},
+        rotation=270,
+        origin={-20,-30})));
+  Modelica.Electrical.Analog.Basic.Ground electricGround2
+    annotation (Placement(transformation(extent={{40,-30},{60,-10}})));
+  Modelica.Electrical.Analog.Sensors.VoltageSensor voltageSensor annotation (
+      Placement(transformation(
+        extent={{-10,10},{10,-10}},
+        rotation=270,
+        origin={60,0})));
+equation
+  connect(core.port_n, measuringCoil.port_p)
+    annotation (Line(points={{10,30},{20,30},{20,10}}, color={255,127,0}));
+  connect(measuringCoil.port_n, airGap.port_p)
+    annotation (Line(points={{20,-10},{20,-50},{10,-50}},
+                                                 color={255,127,0}));
+  connect(electricGround1.p, excitingCoil.n)
+    annotation (Line(points={{-50,-10},{-40,-10}}, color={0,0,255}));
+  connect(rampCurrent.p, electricGround1.p)
+    annotation (Line(points={{-60,-10},{-50,-10}}, color={0,0,255}));
+  connect(rampCurrent.n, excitingCoil.p)
+    annotation (Line(points={{-60,10},{-40,10}}, color={0,0,255}));
+  connect(measuringCoil.n, electricGround2.p)
+    annotation (Line(points={{40,-10},{50,-10}}, color={0,0,255}));
+  connect(measuringCoil.p, voltageSensor.p)
+    annotation (Line(points={{40,10},{60,10}}, color={0,0,255}));
+  connect(electricGround2.p, voltageSensor.n)
+    annotation (Line(points={{50,-10},{60,-10}}, color={0,0,255}));
+  connect(excitingCoil.port_n, magFluxSensor.port_n)
+    annotation (Line(points={{-20,-10},{-20,-20}}, color={255,127,0}));
+  connect(magFluxSensor.port_p, magneticGround.port)
+    annotation (Line(points={{-20,-40},{-20,-50}}, color={255,127,0}));
+  connect(magneticGround.port, airGap.port_n)
+    annotation (Line(points={{-20,-50},{-10,-50}}, color={255,127,0}));
+  connect(excitingCoil.port_p, core.port_p)
+    annotation (Line(points={{-20,10},{-20,30},{-10,30}}, color={255,127,0}));
+  annotation (Documentation(info="<html>
+<p>
+Educational example of a magnetic circuit containing a toroidal iron core with circular cross section and an airgap:
+</p>
+<p>
+A current ramp is applied in positive electric direction through the exciting coil, causing a rising magnetomotive force (mmf) in positive magnetic direction of the electromagnetic converter.  
+The mmf in turn causes a magnetic flux through the circuit in the direction indicated by the flux sensor. 
+From that magnetic flux, flux density can be calculated in every element of the magnetic circuit. Flux density is used to derive magnetic field strength.
+Magnetic field strength times length of the flux line gives magnetic potential difference of each element. 
+The sum of all magnetic potential differences is covered by the mmf of the exciting coil.
```

Comment:
I am not an expert of the technical language in magnetism but could you please double check "is covered by.."?

Reply:
Regarding layout of the toroid: The icons are adapted in an other PR.
Well, "is covered by" means "is equal to". To me it sounds correct. Not quite specific to magnetism. The driving force covers all consumptions.